### PR TITLE
Combine join and create cards into same files

### LINKED ISSUE #52: Add back Join and Create buttons
Created last sprint, deleted from dashboard template. You can edit this one, Eden's or combine both.

New edits:
- separate join and create button to edit colors
 
image

image

### PR DESCRIPTION
**Pull request by mabelene**
closes #52

### CODE REVIEW

File: components/Cards/CreateCard.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+              <Typography variant="body1" className="text-sm pt-1">
+                {description}
+              </Typography>
+            </Box>
+          </div>
+        </div>
+      </Box>
+    </div>
+  );
+}
```
Comment (LemonFace0309) [inline components/Cards/CreateCard.js:34]: did eslint not give you an error for not having an extra line at the end? You turned it on right :3

File: components/Container.js [hunk 2]
```diff
@@ -16,6 +18,18 @@ const Container = () => {
...
+      {/* can be editied to store new cards */}
+      <div className="flex flex-row space-x-5 p-10">
+        <CreateCard
+          spaceName="Create a Space"
+          description="insert some sort of tagline or feature description "
+        />
+        <JoinCard
+          spaceName="Join a Space"
+          description="insert some sort of tagline or feature description "
+        />
```
Comment (LemonFace0309) [inline components/Container.js:31]: These cards are nearly identical. Can you abstract it into one? You can probably add a background prop (ex `light` or `dark`) and conditionally change it in `makeStyles`

**Review by LemonFace0309 — CHANGES_REQUESTED** [review]
Great job overall!

**Review by LemonFace0309 — CHANGES_REQUESTED** [review]
After you changed the file structure the build stopped working. Make sure all your file imports are valid. You can test that by running `npm run build` in the console and checking that everything goes through.

**Comment by eden-chan** [comment]
Not sure how to resolve this.
` npm run build

> ui1@0.1.0 build
> next build

Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db
info  - Creating an optimized production build  
Failed to compile.

ModuleNotFoundError: Module not found: Error: Can't resolve '@material-ui/core/Alert' in '/home/eden/Repo/Study-Space/node_modules/@material-ui/lab/Alert'


> Build error occurred
Error: > Build failed because of webpack errors
    at /home/eden/Repo/Study-Space/node_modules/next/dist/build/index.js:15:918
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async /home/eden/Repo/Study-Space/node_modules/next/dist/build/tracer.js:1:525`

**Comment by LemonFace0309** [comment]
> Not sure how to resolve this.
> ` npm run build
> 
> > ui1@0.1.0 build
> > next build
> 
> Browserslist: caniuse-lite is outdated. Please run:
> npx browserslist@latest --update-db
> info - Creating an optimized production build
> Failed to compile.
> 
> ModuleNotFoundError: Module not found: Error: Can't resolve '@material-ui/core/Alert' in '/home/eden/Repo/Study-Space/node_modules/@material-ui/lab/Alert'
> 
> > Build error occurred
> > Error: > Build failed because of webpack errors
> > at /home/eden/Repo/Study-Space/node_modules/next/dist/build/index.js:15:918
> > at runMicrotasks ()
> > at processTicksAndRejections (internal/process/task_queues.js:95:5)
> > at async /home/eden/Repo/Study-Space/node_modules/next/dist/build/tracer.js:1:525`

Are you using the Alert component? I think it's exported from `@material-ui/lab` or something like that and not `core`. You probably need to adjust imports to match. https://material-ui.com/components/alert/

**Comment by LemonFace0309** [comment]
@eden-chan @mabelene I fixed the deployment. Please merge the create/join spaces card into one now please. Msg me for clarity if you need it.

File: components/Dashboard/Cards/JoinCard.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
```
Comment (LemonFace0309) [inline components/Dashboard/Cards/JoinCard.js:1]: Please delete if we're not gonna use it.

File: components/Dashboard/Cards/DashboardCard.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+            </Typography>
+            <Typography variant="body1" className="text-sm pt-1">
+              {description}
+            </Typography>
+          </Box>
+        </div>
+      </div>
+    </Box>
+  );
+}
```
Comment (LemonFace0309) [inline components/Dashboard/Cards/DashboardCard.js:31]: missing new line

**Review by LemonFace0309 — CHANGES_REQUESTED** [review]

File: components/Dashboard/Cards/DashboardCard.js (hunk 4 above)
Comment (LemonFace0309) [reply components/Dashboard/Cards/DashboardCard.js:31]: make sure you have eslint on. It should've asked you to add PropTypes. It's not added here tho

**Review by LemonFace0309 — APPROVED** [review]

**Pull request merged**